packages/common/src/doctype.ts

```diff
@@ -134,11 +134,11 @@ export abstract class Doctype extends Observable<DocState> implements DocStateHo
     }
 
     get id(): StreamID {
-        return new StreamID(this.state$.value.doctype, this.state$.value.log[0].cid)
+        return new StreamID(this.state$.value.type, this.state$.value.log[0].cid)
     }
 
-    get doctype(): string {
-        return this.state$.value.doctype
+    get doctype(): number {
+        return this.state$.value.type
```

Comment:
I think me and @stbrody discussed this and wanted to keep the doctype as string here for the time being.

Reply:
Oke.

Reply:
Did we discuss this?  I think the only thing we discussed was making sure it stayed in the http client output, which is already handled by the changes to DoctypeUtils.serializeState().

I don't really see any particular reason this *needs* to be included as a string.  But I also don't really feel strongly either way.  I guess it's a little nicer for applications to be able to get the human-readable string name without having to do an extra lookup of StreamType.nameByIndex, so the way it is in the most recent version looks fine. But again, I don't feel strongly either way

Reply:
hm In that case I think we could just remove this getter?

Reply:
Yeah, seems fine to remove.  You can always get the streamtype ID and name from the StreamID